test/cases/ostree-ng.sh

```diff
@@ -340,45 +340,10 @@ sudo qemu-img create -f qcow2 "${LIBVIRT_IMAGE_PATH}" 20G
 greenprint "📑 Generate kickstart file"
 tee "$KS_FILE" > /dev/null << STOPHERE
 text
-lang en_US.UTF-8
-keyboard us
-timezone --utc Etc/UTC
-
-selinux --enforcing
-
-bootloader --timeout=1 --append="net.ifnames=0 modprobe.blacklist=vc4"
-
-network --bootproto=dhcp --device=link --activate --onboot=on
-
 zerombr
 clearpart --all --initlabel --disklabel=msdos
 autopart --nohome --noswap --type=plain
-ostreesetup --nogpg --osname=rhel-edge --remote=rhel-edge --url=file:///ostree/repo --ref=${OSTREE_REF}
-poweroff
-
-%post --log=/var/log/anaconda/post-install.log --erroronfail
-
-# no sudo password for user admin
-echo -e 'admin\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```

Comment:
We need this one for avoid sudo password when run command via ssh.

Reply:
Thanks! I tried fixing this by adding a password instead.